test/lib/functions.test.js

```diff
@@ -213,7 +213,7 @@ describe('FlexFunctions', () => {
     it('should include a body', (done) => {
       const taskName = quickRandom();
       const task = sampleTask(taskName);
-      functions.register(taskName, (context, complete) => complete({ foo: 'bar' }).ok().next());
+      functions.register(taskName, (context, complete) => complete({ foo: 'bar' }).ok().done());
```

Comment:
Should this tests (and all others) be adjusted to use `setBody` or is the older method still valid for backwards compat?